security/design/aliases.go

```diff
@@ -0,0 +1,170 @@
+//************************************************************************//
```

Comment:
Can you remind me this is necessary? The example imports the goa `http` design package - not the security design package. Looking at the security plugin `design` package I don't see a reason it would have to be imported in user designs. I think this file is not necessary (and anything related to it in the Makefile if anything).

Reply:
Doesn't the aliaser automatically create this file for us?

Reply:
Deleted it!